src/OpenDiffix.Core/CommonTypes.fs

```diff
@@ -166,10 +166,13 @@ type AnonymizationParams =
 
 type ExecutorHook = ExecutionContext -> Plan -> seq<Row>
 
+type PostAggregationHook = AggregationContext -> seq<AggregationBucket> -> seq<AggregationBucket>
+
 type QueryContext =
   {
     AnonymizationParams: AnonymizationParams
     DataProvider: IDataProvider
+    PostAggregationHook: PostAggregationHook
```

Comment:
What do you think about making this a list of hooks instead?
It would then be easier to keep the code for LED and *-bucket separate.
Or maybe some experimentation is needed first...

Reply:
I think that can be done with a simple composition in client code. If not, we change it later.